testsuite/tests/singlecluster/gateway/reconciliation/change_targetref/test_update_authpolicy_target_ref.py

```diff
@@ -4,11 +4,22 @@
 
 import pytest
 
+from testsuite.kuadrant.policy.authorization.auth_policy import AuthPolicy
+from testsuite.tests.singlecluster.gateway.reconciliation.change_targetref.conftest import change_target_ref
```

Comment:
```suggestion
from .conftest import change_target_ref
```
will help if there are testsuite structure changes